### PR TITLE
Animation: Fixed bug when using mulitline sprite sheets on macOS.

### PR DESCRIPTION
**Pull request by mattiashagstrand**
Since the y-axis is flipped on macOS compared to iOS the animation for a multiline sprite sheet will start from the bottom row instead of the top row. This PR fixes the issue.

### CODE REVIEW

File: Sources/Core/API/Animation.swift [hunk 1]
```diff
@@ -147,8 +147,14 @@ internal extension Animation {
             return Frame(texture: textures[index], contentRect: contentRect)
         case .spriteSheet(let sheet):
             let framesPerRow = sheet.frameCount / sheet.rowCount
-            let row = index / framesPerRow
-            let column = index - row * framesPerRow
+            let rowIndex = index / framesPerRow
+            let column = index - rowIndex * framesPerRow
+
+            #if os(macOS)
```
Comment (JohnSundell) [inline Sources/Core/API/Animation.swift]: Instead of introducing another variable (which IMO makes the code a bit hard to follow), how about moving this conditional code to when we assign the content rect's origin? So something like:

```swift
#if os(macOS)
contentRect.origin.y = Metric(sheet.rowCount - 1 - row) / Metric(sheet.rowCount)
#else
contentRect.origin.y = Metric(row) / Metric(sheet.rowCount)
#endif
```

Comment (mattiashagstrand) [reply Sources/Core/API/Animation.swift]: Looks good, I'll change it!
